posthog/batch_exports/http.py

```diff
@@ -305,6 +314,15 @@ def validate_destination(self, destination_attrs: dict):
                 raise serializers.ValidationError("Password is required if authentication type is password")
             if config.get("authentication_type") == "keypair" and merged_config.get("private_key") is None:
                 raise serializers.ValidationError("Private key is required if authentication type is key pair")
+        if destination_attrs["type"] == BatchExportDestination.Destination.S3:
+            config = destination_attrs["config"]
+            # TODO: should file format be required?
+            file_format = config.get("file_format")
```

Comment:
It used to not be required as we only did JSONLines, and we still have a few batch exports without file format. So, for backwards compatibility, we kind of have to support it.

Reply:
You can probably assume JSONLines...
```suggestion
            # TODO: should file format be required?
            file_format = config.get("file_format", "JSONLines")
```

Reply:
but yeah, could default to JSONL here